builtin.c

```diff
@@ -740,6 +759,56 @@ Perl_boot_core_builtin(pTHX)
     newXS_flags("builtin::import", &XS_builtin_import, __FILE__, NULL, 0);
 }
 
+/* These pp_ funcs all need to use dXSTARG */
+
+PP(pp_refaddr)
```

Comment:
I feel that, since we have various XSUBs in this file already, the related PP funcs should probably live in the same area, nearer the top, with the wrapping machinary for import/etc at the bottom.

Reply:
moved up